extension/js/common/ui/key-import-ui.ts

```diff
@@ -356,7 +356,19 @@ export class KeyImportUi {
       const keyinfos = await KeyStore.get(acctEmail);
       const privateKeysIds = keyinfos.map(ki => ki.fingerprints[0]);
       if (privateKeysIds.includes(k.id)) {
-        throw new UserAlert('This is one of your current keys, try another one.');
+        const key = keyinfos.find(ki => ki.id === k.id);
+        /* eslint-disable @typescript-eslint/no-non-null-assertion */
+        const existingKey = await KeyUtil.parse(key!.public);
+        const hasNewerExpiration = !!(k.expiration && existingKey.expiration && k.expiration > existingKey.expiration);
+        if (hasNewerExpiration) {
+          const updateKeyMessage =
+            "The key you're trying to import is a newer version of one you already have, based on its expiry date.\n" +
+            'Follow the <a href="https://flowcrypt.com/docs/getting-started/common-tasks/update-private-key.html">guide</a> \n' +
+            'to update it or contact your FlowCrypt admin.';
+          await Ui.modal.info(updateKeyMessage, true);
```

Comment:
Is it possible to add modal button here which will navigate user directly to `update private key` modal:
<img width="600" alt="Screenshot 2025-03-23 at 21 37 31" src="https://github.com/user-attachments/assets/b2cb47d8-ac04-4f84-9af8-ba54e2894f98" />